deploy/crds/app_v1alpha1_kogitoapp_crd.yaml

```diff
@@ -125,6 +125,14 @@ spec:
               description: The name of the runtime used, either quarkus or springboot,
                 defaults to quarkus
               type: string
+            service:
+              description: Service configuration
+              properties:
+                labels:
```

Comment:
Can you please add a description for this property?

Reply:
Description added. Thanks.